### PR TITLE
Fixes bug in WriteToFileStage when configured with a filename without a path

### LINKED ISSUE #48: [BUG] WriteToFileStage fails when given a filename without a path
**Describe the bug**
Initializing WriteToFileStage with:
`WriteToFileStage(config, filename='test.json')`

Will cause a failure in `_build_single`:
`os.makedirs(os.path.dirname(self._output_file), exist_ok=True)`
The call to `os.path.dirname` will return '' causing `os.mkdirs` to raise an exception

### PR DESCRIPTION
**Pull request by dagardner-nv**
fixes #48

### CODE REVIEW

File: morpheus/pipeline/output/to_file.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def _build_single(self, seg: neo.Segment, input_stream: StreamPair) -> StreamPai
             def node_fn(input: neo.Observable, output: neo.Subscriber):
 
                 # Ensure our directory exists
-                os.makedirs(os.path.dirname(self._output_file), exist_ok=True)
+                os.makedirs(os.path.realpath(os.path.dirname(self._output_file)), exist_ok=True)
```
Comment (mdemoret-nv) [inline morpheus/pipeline/output/to_file.py:123]: Isnt this backwards. Shouldnt it be 
```
os.makedirs(os.path.dirname(os.path.realpath(self._output_file)), exist_ok=True)
```

File: tests/conftest.py [hunk 2]
```diff
@@ -167,6 +167,18 @@ def reload_modules(request: pytest.FixtureRequest):
...
 
 
+@pytest.fixture(scope="function")
+def chdir_tmpdir(request: pytest.FixtureRequest, tmp_path):
+    """
+    Executes a test in the tmp_path directory
+    """
+    print(tmp_path)
+    os.chdir(tmp_path)
+    print(1)
```
Comment (mdemoret-nv) [inline tests/conftest.py:177]: Dont need these prints

**Review by mdemoret-nv — CHANGES_REQUESTED** [review]

File: morpheus/pipeline/output/to_file.py (hunk 1 above)
Comment (mdemoret-nv) [reply morpheus/pipeline/output/to_file.py:123]: This leads me to believe that your test might not be working as intended

Comment (dagardner-nv) [reply morpheus/pipeline/output/to_file.py:123]: As far as I can tell the output is always the same, if you prefer one over the other let me know.

```
In [1]: import os

In [2]: os.path.realpath(os.path.dirname('README.md'))
Out[2]: '/home/dagardner/work/morpheus'

In [3]: os.path.realpath(os.path.dirname('README.md')) == os.path.dirname(os.path.realpath('README.md'))
Out[3]: True

In [4]: os.path.realpath(os.path.dirname('./README.md')) == os.path.dirname(os.path.realpath('./README.md'))
Out[4]: True

In [5]: os.path.realpath(os.path.dirname('./examples/log_parsing/README.md')) == os.path.dirname(os.path.realpath('./examples/log_parsin
   ...: g/README.md'))
Out[5]: True
```

File: tests/conftest.py (hunk 2 above)
Comment (dagardner-nv) [reply tests/conftest.py:177]: Well now :) that is embarrassing now isn't it :) 
good catch

**Review by mdemoret-nv — APPROVED** [review]

**Pull request merged**